lib/state/ui/reducer.ts

```diff
@@ -39,6 +37,19 @@ const listTitle: A.Reducer<T.TranslatableString> = (
   }
 };
 
+const showNoteList: A.Reducer<boolean> = (state = false, action) => {
+  switch (action.type) {
+    case 'CLOSE_NOTE': {
+      return true;
+    }
+    case 'App.selectNote':
+      return false;
+
+    default:
+      return state;
+  }
```

Comment:
For actions like `deleteNoteForever` and `restoreNote` and the like is this now handled automatically by some other code, that the note closes?

Reply:
Noted in testing: the note closes automatically due to the filtering middleware which implicitly unselects it _as it no longer exists_.